### PR TITLE
First draft of pollinator indicators

### PR DESCRIPTION
**Pull request by Peprah94**
This is the first draft of the pollinator indicator project. I still have a few things to add to the write-up.

### CODE REVIEW

File: indicators/pollinatorIndicators/R/pollinatorIndicators.qmd [hunk 1]
```diff
@@ -0,0 +1,1013 @@
...
+
+**What does this mean and how do we interpret these thresholds. Does these conditions make sense and how do we interpret those?**
+```{r}
+#| fig-cap: 'Shannon diversity indices at ASO data locations.'
+#| echo: false
+#| warning: false
+#| label: fig-ecosystemCondition
+#| fig-width: 9
+#| fig-height: 9
+  load("../data/ecosystemConditions.RData")
```
Comment (anders-kolstad) [inline indicators/pollinatorIndicators/R/pollinatorIndicators.qmd:412]: This file does not exist.

**Review by anders-kolstad — CHANGES_REQUESTED** [review]
Hi @Peprah94 , I cant render do to one missing RData file. Can you have a look and see if you can locate it?

File: indicators/pollinatorIndicators/R/pollinatorIndicators.qmd (hunk 1 above)
Comment (anders-kolstad) [reply indicators/pollinatorIndicators/R/pollinatorIndicators.qmd:412]: Look here: P:\412421_okologisk_tilstand_2024\pollinatorIndicators\oldScripts\Data\markdownResults\image

**Review by anders-kolstad — CHANGES_REQUESTED** [review]
Thank you @Peprah94 for this PR. The work is very nice. I made direct suggestion in a PR to Peprah:main that you can see [here](https://github.com/Peprah94/ecRxiv/pull/1). I suggest merging this to your main.
The following issues shold be a adressed before moving to the next phase of the review.

1. See comment about concoluted logic using plant ISDMs.
2. Normalisation of Simposon diversity values to create (normative) indicator values

**Comment by anders-kolstad** [comment]
## ASO-data
There should be around 500 ASO meadows with full transect analyses. The data now only contains about 50. Important data are missing.

**Comment by Peprah94** [comment]
> ## ASO-data
> There should be around 500 ASO meadows with full transect analyses. The data now only contains about 50. Important data are missing.

The data we pulled are from GBIF, and the metadata information is formatted. If there are 500 locations, do these 500 locations all contain the data we are interested in? If not, it might not feature in the data. I have done the data format for the ASO meadows like four times.

File: indicators/pollinatorIndicators/R/pollinatorIndicators.qmd [hunk 2]
```diff
@@ -0,0 +1,1013 @@
...
+<hr />
+
+<!--# Document you work below.  -->
+
+## 1. Introduction
+
+<!--# Describe the indicator in general terms. It is a good idea to include a bullet point list of the spesific steps in the workflow -->
+We aim to model the diversity of pollinators in Norway given their know interaction
+with some plant species. From the estimated diversity across the nation, we intend
+to look at the estimates in the ASO Data meadows by using their values as the reference
```
[the remark this answers is not in the document]
Comment (Peprah94) [reply indicators/pollinatorIndicators/R/pollinatorIndicators.qmd]: These are the bits and pieces the other co-authors can help with.

**Review by anders-kolstad — APPROVED** [review]
gg() was switched to geom_sf()

**Pull request merged**